README.md

```diff
@@ -23,6 +23,7 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (optional, required only for LLVM build; add to PATH)
 * [Python](https://www.python.org/downloads/) (optional, required only for LLVM build; add to PATH)
+* [Git] (https://git-scm.com/downloads) (Select "use Git from the Windows Command Prompt")
```

Comment:
git pull --recursive is probably enough anyway, I think by default you have to assume people know how to use git, this is github after all. If not then I doubt they are likely to contribute to the project.